CharacterCreation/Class-Specific Documentation/Engineer Processes.txt

```diff
@@ -222,10 +222,10 @@ Duration: 	engagement
 Range:		Touch
 
     You fiddle with your ally's gun quickly. You tune the whosits, and
-dial the whatsits. It now has a -1 chance to miss and has half the chance of
-jamming round down. Normal use and wear on the gun will almost definitely 
-unalign what you've done by the end of the engagement, but hopefully your 
-efforts will help keep your ally alive until then.
+dial the whatsits. It now has a -1 Miss Chance in all range brackets and has 
```

Comment:
Wait... so... is this ability supposed to _replace_ the miss chance in all range brackets? Because that's how it reads to me, and that would be far too overpowered.